sycl/include/CL/sycl/detail/scheduler/commands.hpp

```diff
@@ -116,10 +117,16 @@ class Command {
 
   std::shared_ptr<event_impl> getEvent() const { return MEvent; }
 
-  virtual ~Command() = default;
-
   virtual void printDot(std::ostream &Stream) const = 0;
 
+  virtual const Requirement *getStoredReq() const {
```

Comment:
getStoredRequirement is too long. Are you OK if I leave it as is?

Reply:
I prefer to proposed version because "Req" is ambiguous w/o the context. At first I thought it's mean "request".
getStoredReq -> "get stored request"?

I don't think getStoredRequirement is "too long", but you can consider making it shorted by removing "Stored" i.e. rename it to "getRequirement". Not sure why having "stored" in the method name might be important.

Reply:
Done.